humility-core/src/hubris.rs

```diff
@@ -3406,14 +3590,14 @@ impl HubrisArchive {
                 };
 
             if userland {
-                for i in 0..=31 {
-                    let reg = match ARMRegister::from_u16(i) {
+                //arm has 32 registers, but riscv has a 33
+                for i in 0..=32 {
```

Comment:
Shouldn't there be a call to get `max_gpr_id` or something similar instead of changing the max value here?

Reply:
Already rewrote how general purpose registers are iterated.  Added a get_all_gp_registers function to help abstract this away.

Reply:
Perfect!